### PR TITLE
Test that output directory exists.

### PR DESCRIPTION
**Pull request by jMonteroMunoz**
I added a small piece of code to check that the output directory exists. Before this, I waited the whole 8 hours time just to see that I had misspelled the output directory and had to restart the script.

### CODE REVIEW

File: gensim/scripts/make_wikicorpus.py [hunk 1]
```diff
@@ -62,6 +62,12 @@
         print(globals()['__doc__'] % locals())
         sys.exit(1)
     inp, outp = sys.argv[1:3]
+    print "Checking if output directory exists: "+'/'.join(outp.split("/")[:-1])
```
Comment (piskvorky) [inline gensim/scripts/make_wikicorpus.py]: No need for this `print` -- just raise the exception.
Also, we'd be using `logger`, not printing.

File: gensim/scripts/make_wikicorpus.py [hunk 2]
```diff
@@ -62,6 +62,12 @@
...
+    if os.path.isdir('/'.join(outp.split("/")[:-1])) == False:
```
Comment (piskvorky) [inline gensim/scripts/make_wikicorpus.py]: Check out `os.path.dirname ` and `os.path.sep` -- this line is not portable.

Also, `== False` is not Pythonic (use `not` operator instead).

Comment (jMonteroMunoz) [inline gensim/scripts/make_wikicorpus.py]: I did the changes you suggested. Thank you for explaining. First time contributing to a project!

**Comment by piskvorky** [comment]
Looks good, thanks @jMonteroMunoz!

@tmylk please take over.

**Comment by tmylk** [comment]
Tested manually and merged in.

Thanks for the PR!

**Comment by piskvorky** [comment]
I suppose we could be doing even more checking (is the outdir writeable? enough free space?)... but if someone needs that, they can open a PR. Probably as a general-purpose utils function in `utils.py`, so we can re-use it in other parts of gensim, in other scripts.

**Comment by jMonteroMunoz** [comment]
I will try to do that whenever I get some time.

**Pull request merged**